### PR TITLE
#451 Dave update docs for consistency

### PR DESCRIPTION
**Pull request by dthiesse**
Did one more pass through Mac and Windows documents to remove any instructions that could be learned from the web sites and to make the wording and format of the documents as consistent as possible. I think these are pretty well good to go now, but will appreciate your thoughtful review and possible merge.

I will go to work on the Linux document next. I do not expect a lot of demand for that in the immediate (next week) term.

### CODE REVIEW

File: setup/set_up_windows.md [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
 2.	In the list of menu options, click __Downloads and Trials__.
 3.	Click __Java for Developers__.
-4.	At the time of writing, the current version of Java JDK 1.8 was under Java SE 8u171. In that section, click the __DOWNLOAD__ button under JDK (Note: your minor version may be different)
-5.	In the section entitled Java SE Development Kit 8u171, click __Accept License Agreement__.
+4.	In the list, look for the newest version that starts with __Java SE 8__. At the time of writing, the newest version was Java SE 8u171. In that section, click the __DOWNLOAD__ button under __JDK__ (Note: your minor version may be different)
+5.	In the section that lists installation files for the various operating systems, click the __Accept License Agreement__ radio button at the top.
 6.	Select the .exe file for Windows x64 (__jdk-8u171-windows-x64.exe__).  The file is downloaded to the machine's Download directory.
 7.	 Run the Windows executable.  Navigate to the Download directory. Double click __jdk-8u171-windows-x64.exe__.
 8.	At the Welcome screen, click __Next__.
 9.	Accept the features to install.  Accept the default install location __C:\Program Files\Java\jdk1.8.0_171\__.  Click __Next__.
```
Comment (palistra) [inline setup/set_up_windows.md]: Looks like some double underscores around __C:\Program Files\Java\jdk1.8.0_171\__

File: setup/set_up_windows.md [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
 10.	Upon successful installation, click __Close__.
 11.	Verify that the installation was successful.  Open a command prompt.  Change directories to __C:\Program Files\Java\jdk1.8.0_171\__. You should see 6 directories, 6 files, and 2 zip files.
```
Comment (palistra) [inline setup/set_up_windows.md]: Looks like some double underscores around  __C:\Program Files\Java\jdk1.8.0_171\__

**Review by palistra — CHANGES_REQUESTED** [review]
Couple of lines with extra underscores - I tried to change but not sure I was doing it right so I am kicking it back to you :-D.

**Comment by dthiesse** [comment]
Thanks Jim. I'll have another look at the PDF for the Windows document.

**Comment by dthiesse** [comment]
It looks like my second commit to fix the formatting problems in the Windows file is included in this pull request. It was a trailing backslash in 2 Windows paths that was causing the problem.

**Review by palistra — APPROVED** [review]

**Review by palistra — APPROVED** [review]

**Pull request merged**